api/exchange.go

```diff
@@ -14,6 +14,18 @@ type ExchangeAPIKey struct {
 	Secret string
 }
 
+// CcxtParam specifies an additional ccxt parameter
+type CcxtParam struct {
+	Parameter string
+	Value     string
+}
+
+// ExchangeHeader specifies additional HTTP headers for centralized exchanges
```

Comment:
same logic as comment above, this comment should not say "centralized exchanges" since it abstracts away the concept of the underlying exchange.